elements/pfe-label/demo/pfe-label.html

```diff
@@ -9,14 +9,12 @@ <h2 slot="header">Default</h2>
     </div>
 </pfe-band>
 
-<pfe-band color="lightest" size="small">
+<pfe-band color-palette="lightest" size="small">
     <h2 slot="header">Color Options</h2>
     <pfe-label color="blue">Blue label</pfe-label>
     <pfe-label color="green">Green label</pfe-label>
     <pfe-label color="orange">Orange label</pfe-label>
-    <pfe-label color="red">
-    Red <span class="visually-hidden-class">Hat</span>&nbsp;label
-    </pfe-label>
+    <pfe-label color="red">Red <span class="visually-hidden-class">Hat</span> label</pfe-label>
```

Comment:
Removing the `&nbsp;` from this line causes the label to collapse on itself to `Redlabel` instead of `Red label` to fix wrap the change in another span.
```Suggestion
    <pfe-label color="red"><span>Red <span class="visually-hidden-class">Hat</span> label</span></pfe-label>
```